### PR TITLE
Access control - Ability to hide user identities

### PR DESCRIPTION
**Pull request by mannyrivera2010**
In Reference to #201 
Please Review
@clarkjefcoat 
@akonsowski 
@rkenyon1969

### CODE REVIEW

**Comment by mannyrivera2010** [comment]
To test, go to anonymize_identifiable_data function in plugins\default_access_control\main.py to True and clear the cache

File: ozpcenter/api/listing/serializers.py [hunk 1]
```diff
@@ -209,6 +221,18 @@ class Meta:
...
 
+    def to_representation(self, data):
+        ret = super(CreateListingProfileSerializer, self).to_representation(data)
+
+        # Used to anonymize usernames
+        anonymize_identifiable_data = system_anonymize_identifiable_data(self.context['request'].user.username)
+
+        if anonymize_identifiable_data:
+            ret['display_name'] = '*'
+            ret['display_name'] = '*'
```
Comment (clarkjefcoat) [inline ozpcenter/api/listing/serializers.py]: Duplicate line?

Comment (mannyrivera2010) [inline ozpcenter/api/listing/serializers.py]: I will remove the extra line

**Comment by clarkjefcoat** [comment]
:+1:

**Comment by mannyrivera2010** [comment]
Issue: When login in as jones, and anonymize_identifiable_data = True, it shows the administration tab.

**Comment by akonsowski** [comment]
:+1:

**Pull request merged**